src/utils/validateBurnedAmount/validateStaticBurnedAmount.test.ts

```diff
@@ -16,26 +16,14 @@ import {
   newBaseTx as pvmBaseTx,
   newExportTx as pvmExportTx,
   newImportTx as pvmImportTx,
-  newAddValidatorTx,
-  newAddDelegatorTx,
-  newCreateSubnetTx,
-  newCreateBlockchainTx,
-  newAddSubnetValidatorTx,
-  newTransformSubnetTx,
-  newAddPermissionlessValidatorTx,
-  newAddPermissionlessDelegatorTx,
   newRemoveSubnetValidatorTx,
   newTransferSubnetOwnershipTx,
```

Comment:
Should we remove all pvm tx types from this utility and test? Same comment as below... Post etna there are no static fee pvm txs

Reply:
In `validateBurnedAmount` there is a reference to Etna. Do we want to remove that reference as well, so we always use the `validateDynamicBurnedAmount` for p chain txs?